<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainClass</class>
 <widget class="QMainWindow" name="CMainClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMain</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbxPhotoAlbum">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>225</y>
      <width>300</width>
      <height>37</height>
     </rect>
    </property>
    <property name="title">
     <string>gbxPhotoAlbum</string>
    </property>
    <widget class="QLabel" name="lblPhotoSize">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>12</y>
       <width>169</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>lblPhotoSize</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtnPhotoAlbumShow">
     <property name="geometry">
      <rect>
       <x>268</x>
       <y>12</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblPhoto">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>12</y>
      <width>300</width>
      <height>205</height>
     </rect>
    </property>
    <property name="text">
     <string>lblPhoto</string>
    </property>
   </widget>
   <widget class="QTableView" name="tabvInfo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>4</y>
      <width>565</width>
      <height>517</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxShortInfo">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>268</y>
      <width>300</width>
      <height>249</height>
     </rect>
    </property>
    <property name="title">
     <string>gbxShortInfo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
